target/linux/realtek/dts-5.10/rtl8380_netgear_gs308t-v1.dts

```diff
@@ -1,8 +1,34 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 #include "rtl8380_netgear_gigabit_3xx.dtsi"
+#include <dt-bindings/leds/common.h>
 
 / {
 	compatible = "netgear,gs308t-v1", "realtek,rtl838x-soc";
 	model = "Netgear GS308T v1";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_amber;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_amber;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_amber: led-0 {
+			label = "amber:status";
+			color = <LED_COLOR_ID_AMBER>;
+			function = LED_FUNCTION_STATUS;
```

Comment:
Just a minor question: how is the LED labelled on the device? You use "status" in the here, but "power" in the commit message.

Reply:
It's a bi-color Power LED, so yes: it should be called power. I overlooked that when copying it from the GS110 (late at night or better: early in the morning).

Reply:
(judging from the GS110TP product images, the LED is also called "Power" on that model)

Reply:
I'm fine with either LED_FUNCTION_STATUS, or LED_FUNCTION_POWER; their documentation appears to allow both to be used for status indication. (Maybe 'status' for a default off, and 'power' for a default on LED?).

Since the device has it labelled as 'power', that's probably what we should go for here. Can you make the node label, label property, and function property consistent still?

Reply:
I went for LED_FUNCTION_POWER because of the label on the device. This function property seem relatively new, I can't remember having it added to any of the other DTS's I contributed?

Reply:
Looks like it was added two years ago, with linux commit 4b37883a8c07c7c7ec166ad5b66d8d0f39e7994d. So only available in OpenWrt since 5.10. I don't think we have a policy (yet) to use color and function properties on every new device either.